Driver/Driver/Extenison/JailbreakChecker.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  JailbreakChecker.swift
+//  Driver
+//
+//  Created by Turan Çabuk on 26.02.2023.
+//
+
+import Foundation
+
+import UIKit
```

Comment:
UIKit 2 kez, Foundation 1 kez tanımlanmış. UIKit'in içerisinde Foundation mevcut. Bu sebepten ötürü sadece
`import UIKit` yeterli bizim için.